frontend/src/components/shared/ErrorPanel.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <transition name="slide-fade">
+    <article v-if="message" id="errorPanel" class="message is-danger">
+      <div class="message-header">
+        <b>{{ title }}</b>
+        <button v-if="hasHideListener" class="delete" aria-label="delete" @click="$emit('hideErrorPanel')" />
+      </div>
+      <div class="message-body has-text-centered">
+        <h5 class="title is-6" v-html="message" />
```

Comment:
There's an asymmetric space above the error message, so I'm suggesting
```suggestion
        <p class="title is-6" v-html="message" />
```

Reply:
Great change and thanks for making it convenient with the "Commit suggestion" feature!